src/modules/jcms_rest/tests/src/Functional/RecursiveEndpointValidatorTest.php

```diff
@@ -56,90 +56,88 @@ public function setUp() {
    */
   public function dataProvider() : array {
     return [
-      [
+      '/subjects' => [
```

Comment:
Changes to this test fixture were made after repeatedly running it against content imported from a recent database dump. This ensures that better feedback is available for failing tests.